mod/dfrn_request.php

```diff
@@ -806,7 +812,7 @@ function dfrn_request_content(&$a) {
 			$myaddr = $_GET['address'];
 		elseif(local_user()) {
```

Comment:
Standards: Can you please review this whole conditional structure for brackets and spaces? 😊